client/src/App.js

```diff
@@ -13,18 +13,15 @@ export default class App extends Component {
   }
 
   addToSavedList = movie => {
-    console.log("savedList:", this.state.savedList);
-    console.log("movie:", movie);
-    console.log("state check", this.state.savedList.includes(movie));
     const savedList = this.state.savedList;
-    console.log("var check", savedList.includes(movie));
+    const stringList = savedList.map(item => JSON.stringify(item));
+    const stringMovie = JSON.stringify(movie);
     // Added logic so the user cannot save a movie twice:
-    if (savedList.includes(movie)) {
+    if (stringList.includes(stringMovie)) {
```

Comment:
another clean way of doing this that still throws an alert to the user if attempting to save the same movie twice. very well done